src/Tasks/Backup/BackupJob.php

```diff
@@ -227,7 +227,9 @@ protected function createZipContainingEveryFileInManifest(Manifest $manifest): s
 
         consoleOutput()->info("Created zip containing {$zip->count()} files and directories. Size is {$zip->humanReadableSize()}");
 
-        $this->sendNotification(new BackupZipWasCreated($pathToZip));
+        $this->encryptBackup($pathToZip);
+
+        $this->sendNotification($pathToZip);
```

Comment:
Why `$this->sendNotification($pathToZip);`?

Reply:
You are right, the flow I worked out seems redundant. I try solving this later and come up with a better solutioN.

Reply:
Just delete that line

Reply:
I will do that today. Sorry for the delay, I've been busy and I forgot about this!